contrib/babelfishpg_tsql/src/pltsql_partition.c

```diff
@@ -202,6 +199,8 @@ bbf_create_partition_tables(CreateStmt *stmt)
 	 */
 	if (partition_column_typoid != input_type_oid)
 	{
+		systable_endscan(scan);
+		table_close(rel, AccessShareLock);
```

Comment:
I think we can skip this. Error handling clean up should be able to handle it.

Reply:
example [ref in engine](https://github.com/babelfish-for-postgresql/postgresql_modified_for_babelfish/blob/BABEL_4_X_DEV__PG_16_X/src/backend/utils/cache/relcache.c#L1739)